backend/src/db/seed.js

```diff
@@ -472,24 +478,28 @@ const languages = ['de', 'en', 'es', 'fr', 'it', 'pt', 'pl']
         'post',
         {
           id: 'p10',
-          imageBlurred: true,
         },
         {
           categoryIds: ['cat10'],
           author: dewey,
+          image: Factory.build('image', {
+            sensitive: true,
```

Comment:
> [<img alt="alina-beck" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/alina-beck) **Authored by [alina-beck](https://github.com/alina-beck)**
_<time datetime="2020-02-18T15:46:11Z" title="Tuesday, February 18th 2020, 4:46:11 pm +01:00">Feb 18, 2020</time>_

---
> **Outdated (history rewrite)** - original diff
---
```diff
@@ -472,24 +478,28 @@ const languages = ['de', 'en', 'es', 'fr', 'it', 'pt', 'pl']
         'post',
         {
           id: 'p10',
-          imageBlurred: true,
         },
         {
           categoryIds: ['cat10'],
           author: dewey,
+          image: Factory.build('image', {
+            blurred: true,
```


can we have images without `url`?

Reply:
> [<img alt="alina-beck" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/alina-beck) **Authored by [alina-beck](https://github.com/alina-beck)**
_<time datetime="2020-02-18T15:46:33Z" title="Tuesday, February 18th 2020, 4:46:33 pm +01:00">Feb 18, 2020</time>_

---


and `aspectRatio`? :grin:

Reply:
> [<img alt="mattwr18" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/mattwr18) **Authored by [mattwr18](https://github.com/mattwr18)**
_<time datetime="2020-02-20T19:40:29Z" title="Thursday, February 20th 2020, 8:40:29 pm +01:00">Feb 20, 2020</time>_

---


I don't think so!?

Reply:
> [<img alt="roschaefer" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/roschaefer) **Authored by [roschaefer](https://github.com/roschaefer)**
_<time datetime="2020-02-22T12:38:31Z" title="Saturday, February 22nd 2020, 1:38:31 pm +01:00">Feb 22, 2020</time>_

---


`Factory.build('image', ...) ` makes sure that there are default values when you don't explicitly override them when you call the factory.